src/cljs/rems/focus.cljs

```diff
@@ -2,13 +2,18 @@
   "Focuses an HTML element as soon as it exists.")
 
 (defn focus-element-async
-  ([selector]
-   (focus-element-async selector 100))
-  ([selector tries]
-   (when (pos? tries)
-     (if-let [element (.querySelector js/document selector)]
-       (do
-         (.setAttribute element "tabindex" "-1")
-         (.focus element))
-       (js/setTimeout #(focus-element-async selector (dec tries))
-                      10)))))
+  "Focus an element when it appears. Options can include:
+    :tries -- number of times to poll, defaults to 100
+    :scroll? -- whether to .scollIntoView the element"
+  [selector & [options]]
+  (let [tries (get options :tries 100)
+        scoll? (get options :scroll? false)]
```

Comment:
There's a typo here. It seems to work anyway, which is kind of funny & strange. However, I assume that having :scroll? as true wouldn't work.